com.bardsoftware.constable/client/src/main/kotlin/com/bardsoftware/constable/client/ConstableClient.kt

```diff
@@ -0,0 +1,33 @@
+package com.bardsoftware.constable.client
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.arguments.argument
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import mu.KotlinLogging
+import java.sql.SQLException
+
+private val logger = KotlinLogging.logger {}
+
+class ConstableClient : CliktCommand() {
+    private val path by argument(name = "h2-path", help = "Path to database file")
+    private val isQuery by option("-q",
+            help = "Specify if SQL command is a query and an output is required").flag(default = false)
+    private val command by option("--h2-command", help = "SQL command to execute").required()
+
+    override fun run() {
+        try {
+            if (isQuery)
```

Comment:
You can use the same `execute` for everything. Its boolean return value indicates if there is a result set (use `getResultSet` to get it)
https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#execute(java.lang.String).